### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by odysseuskir**
Solves #8

### CODE REVIEW

**Comment by odysseuskir** [comment]
Solves #8

**Comment by GalaxyGamingBoy** [comment]
Seems OK BUT I would prefer some changes

File: README.md [hunk 1]
```diff
@@ -1,42 +1,7 @@
...
-```
-
-Locally preview production build:
-
-```bash
-npm run preview
-```
-
-Check out the [deployment documentation](https://nuxt.com/docs/getting-started/deployment) for more information.
+Test Wiz is a web application that allows teachers to create and manage tests for their students. Teachers can create tests with multiple choice, true/false, and short answer questions. Students can then take these tests and see their results.
```
Comment (GalaxyGamingBoy) [inline README.md:7]:
```suggestion
Test Wiz is a web application that allows teachers to create and manage tests for their students. Teachers can create tests with multiple choice, true/false, and short answer questions. Students can then take these tests and see their results.

```
Please add a new line at the end

Comment (GalaxyGamingBoy) [inline README.md:7]:
```suggestion
Test Wiz is a web application that allows teachers to create and manage tests for their students. Teachers can create tests with multiple choice, true/false, and short answer questions. Students can then take these tests and see their results.

```
Please add a new line at the end

**Review by GalaxyGamingBoy — CHANGES_REQUESTED** [review]
Also I would add more Sections like:
+ How Testwiz Works
+ AI integrated*

**Comment by odysseuskir** [comment]
Will do!

**Comment by GalaxyGamingBoy** [comment]
Yep, ty!
When is your exam?

**Comment by odysseuskir** [comment]
Done! I have to be at the test center at 17:00. I will be examined at 17:30 - 18:00, it is an oral exam.

**Comment by GalaxyGamingBoy** [comment]
LGTM!

**Review by GalaxyGamingBoy — APPROVED** [review]
LGTM! Please merge

**Comment by odysseuskir** [comment]
MERGED!

**Comment by GalaxyGamingBoy** [comment]
YAY!

**Pull request merged**